client/src/redux/reducers/allocations.reducers.js

```diff
@@ -75,9 +81,23 @@ function allocations(state = initialState, action) {
         ...state,
         search: { ...state.search, results: action.payload.data }
       };
+    case 'ALLOCATION_OPERATION_REMOVE_USER_INIT': {
```

Comment:
Not super important, but I was thinking about this verbosity. Do you think it's better than something like `ALLOCATION_REMOVE_USER`?

Reply:
Yes. good idea